DownFall/Extensions/CGPoint+Extension.swift

```diff
@@ -197,4 +211,9 @@ extension CGPoint {
         case center
         case bottom
     }
+    
+    func translateVertically(_ yOffset: CGFloat) -> CGPoint {
+        return CGPoint(x: self.x, y: self.y + yOffset)
+    }
```

Comment:
We should use a function like this to help the static methods. It would help avoid typos